README.md

```diff
@@ -41,29 +41,36 @@ to operate on. There are many providers for the
 interface. e.g.
 
 ```js
-   var ram = require('random-access-memory')
-   var feed = hyperdb(function (filename) {
-     // filename will be one of: data, bitfield, tree, signatures, key, secret_key
-     // the data file will contain all your data concattenated.
-
-     // just store all files in ram by returning a random-access-memory instance
-     return ram()
-   })
+var ram = require('random-access-memory')
+var feed = hyperdb(function (filename) {
+  // filename will be one of: data, bitfield, tree, signatures, key, secret_key
+  // the data file will contain all your data concattenated.
+
+  // just store all files in ram by returning a random-access-memory instance
+  return ram()
+})
 ```
 
 `key` is a `Buffer` containing the local feed's public key. If you do not set
 this the public key will be loaded from storage. If no key exists a new key pair
 will be generated.
 
+Options include:
+
+```js
+{
+  map: node => mappedNode, // map nodes before returning them
+  reduce: (a, b) => someNode, // reduce the nodes array before returning it
+  one: false, // set to true to reduce the array to a single node
```

Comment:
What does this mean? I read this as "don't call `reduce` unless `one` is truthy" which maybe wasn't your intention.